Packs/RecordedFutureASI/README.md

```diff
@@ -0,0 +1,10 @@
+# Pack Documentation
```

Comment:
Documentation shouldnt be the Pack readme

Reply:
done